docs/before/accounts.md

```diff
@@ -0,0 +1,101 @@
+
+!!! note "Quicklinks"
+    - **Account** - [Create a UNIX account][6]
+    - **Computing** - [Access and Resources][1]
+    - **Data Analysis** - for [LCLS-I][2] and for [LCLS-II][3]
+    - **FFB docs** - [PCDS docs][8]
+    - **S3DF docs** - ["Running at S3DF" for LCLS][4] and [official S3DF documentation][5].
+
+
+# Step 1 - Get a computer account
+
+To access SLAC computing resources, you need a valid SLAC UNIX account. A detailed description of the steps required to create one can be found [here][6].
+
+!!! note "Enabling access to the S3DF"
+    In order to use S3DF, it is necessary to enable your S3DF account using your SLAC UNIX account. Detailed information [in the S3DF documentation here][7].
+
+!!! danger "Access to NERSC"
+    At the moment, users need a NERSC account to run LCLS workflows there. See [this link][9] for some information on how to create an account at NERSC. Work is in progress to remove this unncessary constraint.
+
+# Step 2 - Access and process data
+Now that you can access the computer systems at SLAC, it is time to get yourself oriented. 
+It is useful for that to understand how the data flows from the beamline to storage, *a.k.a* to understand the LCLS Data Systems, built with 3 successive levels:
+
+- **1 &#x3e;  DAQ+DRP - Data acquisition and reduction**:<br>
+provides diagnostics within <1s. *(Experts only)*
+- **2 &#x3e;  FFB - Fast Feedback**:<br> 
+provides diagnostics and results within <1min. *(All)*
+- **3 &#x3e;  Offline Analysis**:<br> 
+for intensive compute during and after the beamtime. *(All)*
+
+*TODO: figure to describe the network and data infrastructure.*
+
+!!! danger "FFB migration to S3DF"
+    S3DF is a new capability at SLAC and is currently still a work in progress. 
+    The FFB nodes will soon be a part of it - when that happens, it will be more seamless to switch from online analysis to offline analysis at the end of the beamtime. 
```

Comment:
and between shifts as well? I also wonder if it would be useful at some point on this page to note that each experiment is assigned a particular queue on the FFB during the experiment, while anaq should be used between shifts and after?

Reply:
Good idea. Let's follow Gabriel's suggestion and have another page about batch and interactive computing with SLURM and Jupyter.